eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState.hs

```diff
@@ -995,9 +1047,11 @@ reapRewards dStateRewards withdrawals =
 -- epoch boundary calculations --
 ---------------------------------
 
+-- | Compute the current Stake Distribution. This was called at the Epoch boundary in the Snap Rule.
```

Comment:
If it's only used in tests, can we move it to a testing package?

Reply:
It originally appeared only in tests. But is now used in two places were things need to be initialized correctly. So it is usefull here both because it is used, and it provides documentation.